packages/react/src/jsx/ReactJSXElement.js

```diff
@@ -352,15 +377,31 @@ export function jsxProdSignatureRunningInDevWithDynamicChildren(
     const trackActualOwner =
       __DEV__ &&
       ReactSharedInternals.recentlyCreatedOwnerStacks++ < ownerStackLimit;
+    let debugStackDEV = false;
+    if (__DEV__) {
+      if (trackActualOwner) {
+        if (supportsSettingStackTraceLimit) {
+          try {
+            const previousStackTraceLimit = Error.stackTraceLimit;
+            Error.stackTraceLimit = ownerStackTraceLimit;
+            debugStackDEV = Error('react-stack-top-frame');
+            Error.stackTraceLimit = previousStackTraceLimit;
+          } catch {}
+        }
+        if (!debugStackDEV) {
+          debugStackDEV = Error('react-stack-top-frame');
+        }
+      } else {
+        debugStackDEV = unknownOwnerDebugStack;
+      }
+    }
```

Comment:
Same approach everywhere but I didn't create a helper to save a stackframe. gcc doesn't have a `@inline` directive afaict. Only `@noinline`